google/datalab/storage/commands/_storage.py

```diff
@@ -52,16 +52,16 @@ def _extract_storage_api_response_error(message):
 
 
 @IPython.core.magic.register_line_cell_magic
-def storage(line, cell=None):
-  """Implements the storage cell magic for ipython notebooks.
+def gcs(line, cell=None):
+  """Implements the gcs cell magic for ipython notebooks.
 
   Args:
-    line: the contents of the storage line.
+    line: the contents of the gcs line.
   Returns:
     The results of executing the cell.
   """
-  parser = google.datalab.utils.commands.CommandParser(prog='storage', description="""
-Execute various storage-related operations. Use "%storage <command> -h"
+  parser = google.datalab.utils.commands.CommandParser(prog='gcs', description="""
+Execute various GCS-related operations. Use "%gcs <command> -h"
```

Comment:
In this context (the help message), we should expand "GCS" into "Google Cloud Storage"